connector/failoverconnector/README.md

```diff
@@ -0,0 +1,79 @@
+
+[in development]:https://github.com/open-telemetry/opentelemetry-collector#in-development
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+
+## Supported Pipeline Types
+
+| [Exporter Pipeline Type] | [Receiver Pipeline Type] | [Stability Level] |
+| ------------------------ | ------------------------ |-------------------|
+| traces | traces | [In development]  |
+| metrics | metrics | [In development]           |
+| logs | logs | [In development]           |
+
+[Exporter Pipeline Type]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/connector/README.md#exporter-pipeline-type
+[Receiver Pipeline Type]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/connector/README.md#receiver-pipeline-type
+[Stability Level]: https://github.com/open-telemetry/opentelemetry-collector#stability-levels
+<!-- end autogenerated section -->
+
+Allows for health based routing between trace, metric, and log pipelines depending on the health of target downstream exporters.
+
+## Configuration
+
+If you are not already familiar with connectors, you may find it helpful to first visit the [Connectors README].
+
+The following settings are available:
+
+- `priority (required)`: list of pipeline level priorities in a 1 - n configuration, multiple pipelines can sit at a single priority level.
+- `retry_interval (required)`: the frequency at which the pipeline levels will attempt to reestablish connection with all higher priority levels
+- `retry_gap (required)`: the amount of time between trying two separate priority levels in a single retry_interval timeframe
+- `max_retry (required)`: the maximum retries per level
+
+Example:
+
+```yaml
+connectors:
+  failover:
+    priority:
+      - [traces/first, traces/fourth]
+      - [traces/second]
+      - [traces/third]
+    retry_pipeline_interval: 5m
+    retry_gap: 1m
+    max_retry: 10
+
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      exporters: [failover]
+    traces/first:
+      receivers: [failover]
+      exporters: [otlp/first]
+    traces/second:
+      receivers: [failover]
+      exporters: [otlp/second]
+    traces/third:
+      receivers: [failover]
+      exporters: [otlp/third]
+    traces/fourth:
+      receivers: [failover]
+      exporters: [otlp/fourth]
+```
+
+The connector intakes a list of 'levels' each of which can contain multiple pipelines. If any pipeline at a stable level fails, the level is considered unhealthy and the connector will move down one priority level and route all data to the new level (assuming it is stable).
```

Comment:
Where possible, it's helpful to use the exact config keys so the user doesn't have to assume what is intended.
```suggestion
The connector intakes a list of `priority_levels` each of which can contain multiple pipelines. If any pipeline at a stable level fails, the level is considered unhealthy and the connector will move down one priority level and route all data to the new level (assuming it is stable).
```

Reply:
Addressed